mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/components/SlippageToleranceV2.tsx

```diff
@@ -51,20 +50,16 @@ export function SlippageToleranceV2({
   const [selectedSlippage, setSelectedSlippage] = useState(slippage.toFixed(8));
   const [isRiskWarningDisplayed, setIsRiskWarningDisplayed] = useState(false);
 
-  const isCustomValue = !Object.values(SlippageAmountButtonTypes).some(
-    (buttonAmount) =>
-      new BigNumber(new BigNumber(buttonAmount).toFixed(8)).isEqualTo(
-        selectedSlippage
-      )
-  );
-  const [isCustomAmount, setIsCustomAmount] = useState(isCustomValue);
-
-  const submitSlippage = debounce(setSlippage, 500);
-  const onSlippageChange = (value: string): void => {
-    setSelectedSlippage(new BigNumber(value).toFixed(8));
-    submitSlippage(new BigNumber(value), isCustomValue);
+  const isCustomValue = (slippage: string) => {
```

Comment:
can help rename this, maybe checkIfCustomSlippage? isCustomValue sounds like it's just a boolean and not a function